src/applications/disability-benefits/all-claims/config/schema.js

```diff
@@ -717,6 +718,49 @@ const schema = {
         ],
       },
     },
+    ptsdIncident: {
+      type: 'object',
+      properties: {
+        date: { $ref: '#/definitions/date' },
+        description: { type: 'string' },
+      },
+    },
+    secondaryPtsdIncident: {
+      type: 'object',
+      properties: {
+        authorities: {
+          type: 'array',
+          items: {
+            type: 'object',
+            properties: {
+              name: {
+                type: 'string',
+              },
+              address: {
+                type: 'object',
+                required: [],
```

Comment:
There's a bug in the address ui schema generator that requires this to be present